Frontend/src/pages/registration_edit/components/RegistrationEditor.jsx

```diff
@@ -287,6 +288,31 @@ export default function RegistrationEditor() {
               />
             </>
           )}
+          <Header>Registration History</Header>
+          <Table>
+            <Table.Header>
+              <Table.Row>
+                <Table.HeaderCell>Timestamp</Table.HeaderCell>
+                <Table.HeaderCell>Changes</Table.HeaderCell>
+                <Table.HeaderCell>Acting User</Table.HeaderCell>
+              </Table.Row>
+            </Table.Header>
+            <Table.Body>
+              {registration.history.map((entry) => (
+                <Table.Row key={entry.timestamp}>
+                  <Table.Cell>{getShortDateString(entry.timestamp)}</Table.Cell>
```

Comment:
For consistency with other uses of short date strings in tables:
```suggestion
      <Table.Cell>
        <Popup
          content={getShortTimeString(registered_on)}
          trigger={<span>{getShortDateString(registered_on)}</span>}
        />
      </Table.Cell>
```